src/events/components/SearchModule/dateRangeInput.css

```diff
@@ -0,0 +1,21 @@
+.InputFromTo .DayPicker-Day--selected:not(.DayPicker-Day--start):not(.DayPicker-Day--end):not(.DayPicker-Day--outside) {
+    background-color: #f0f8ff !important;
+    color: #4a90e2;
+}
+.InputFromTo .DayPicker-Day {
+    border-radius: 0 !important;
+}
+.InputFromTo .DayPicker-Day--start {
+    border-top-left-radius: 50% !important;
+    border-bottom-left-radius: 50% !important;
+}
+.InputFromTo .DayPicker-Day--end {
+    border-top-right-radius: 50% !important;
+    border-bottom-right-radius: 50% !important;
+}
+.InputFromTo .DayPickerInput-Overlay {
+    width: 550px;
+}
+.InputFromTo-to .DayPickerInput-Overlay {
+    margin-left: -198px;
+}
```

Comment:
These styles could maybe be moved to a Less-document, but only `.InputFromTo` is defined by `DateRangeInput`, and as such the classes don't have the hashes at the end, which are added when using less, so we would need to find a way for the other classes to not have added hashes

Reply:
[You might be looking for :global](https://github.com/css-modules/css-modules#usage-with-preprocessors)

Reply:
Would _very_ much like for this to be a Less file, not CSS.
It would be a bit more readable it it is nested IMHO